src/cascadia/TerminalSettingsEditor/Appearances.xaml

```diff
@@ -286,6 +288,57 @@
                     </Grid>
                 </StackPanel>
             </local:SettingContainer>
+            <local:SettingContainer x:Uid="Profile_FontAxes"
+                                    Style="{StaticResource ExpanderSettingContainerStyle}">
+                <StackPanel Spacing="16">
+                    <ListView IsItemClickEnabled="False"
+                              ItemsSource="{Binding Source={StaticResource FontAxesCVS}}"
+                              SelectionMode="None">
+                        <ListView.Resources>
+                            <CollectionViewSource x:Key="FontAxesNamesCVS"
+                                                  x:Name="FontAxesNamesCVS" />
+                        </ListView.Resources>
+                        <ListView.ItemTemplate>
+                            <DataTemplate x:DataType="local:AxisKeyValuePair">
+                                <Grid Margin="-16,0,-12,0"
+                                      ColumnSpacing="8">
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="Auto" />
+                                        <ColumnDefinition Width="Auto" />
+                                        <ColumnDefinition Width="*" />
+                                    </Grid.ColumnDefinitions>
+                                    <ComboBox Grid.Column="0"
+                                              IsSynchronizedWithCurrentItem="False"
+                                              ItemsSource="{Binding Source={StaticResource FontAxesNamesCVS}}"
+                                              SelectedIndex="{x:Bind AxisIndex, Mode=TwoWay}">

```

Comment:
This is an unfortunate xaml workaround. XAML seems to struggle with ComboBox bindings - i.e. we cannot just bind `AxisKey` to the ComboBox's `SelectedValue`, `SelectedItem` or even `Text` in a two-way manner. So we bind to `SelectedIndex` instead and do some manual conversion for what key the index maps to.